manifest.json

```diff
@@ -30,7 +30,7 @@
         "modal.html"
     ],
     "content_scripts": [{
-        "js": ["libs/jquery.min.js", "libs/extLibs.js", "main.js"],
+        "js": ["libs/jquery.min.js", "libs/extLibs.js", "libs/virtual-key-code.min.js", "main.js"],
```

Comment:
@J2TeaM :+1: I thought it was a library >.<, just is constans so ok,and the next, allow users config their hotkeys is great furute, we have to it to next update :)